INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:04:40; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:04:40; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:40; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:40; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:41; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:41; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:41; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:41; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:41; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:41; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:41; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:41; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:41; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:41; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:42; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:42; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:42; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:42; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:42; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:42; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:42; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:42; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:42; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:42; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:42; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:42; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:43; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:43; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:43; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:43; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:43; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:43; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:06:43; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:06:43; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 3219 (2018-07-06 14:07:47; basehttp.py:124)
INFO:django.server: "POST /done/17/ HTTP/1.1" 302 0 (2018-07-06 14:07:49; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 2900 (2018-07-06 14:07:49; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 10592 (2018-07-06 14:07:53; basehttp.py:124)
INFO:django.server: "GET /16/ HTTP/1.1" 200 2897 (2018-07-06 14:08:15; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 10592 (2018-07-06 14:08:18; basehttp.py:124)
INFO:django.server: "GET /17/ HTTP/1.1" 200 2900 (2018-07-06 14:08:20; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 10592 (2018-07-06 14:08:21; basehttp.py:124)
INFO:django.server: "GET /15/ HTTP/1.1" 200 3215 (2018-07-06 14:08:22; basehttp.py:124)
INFO:django.server: "POST /done/15/ HTTP/1.1" 302 0 (2018-07-06 14:08:27; basehttp.py:124)
INFO:django.server: "GET /15/ HTTP/1.1" 200 2896 (2018-07-06 14:08:27; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 10592 (2018-07-06 14:08:33; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 10592 (2018-07-06 14:12:27; basehttp.py:124)
INFO:django.server: "GET /notice/new/ HTTP/1.1" 200 3245 (2018-07-06 14:12:29; basehttp.py:124)
INFO:django.server: "POST /notice/new/ HTTP/1.1" 302 0 (2018-07-06 14:12:37; basehttp.py:124)
INFO:django.server: "GET /18/ HTTP/1.1" 200 2914 (2018-07-06 14:12:37; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 11058 (2018-07-06 14:12:38; basehttp.py:124)
INFO:django.server: "GET /add_money/2/ HTTP/1.1" 200 3102 (2018-07-06 14:12:40; basehttp.py:124)
INFO:django.server: "POST /add_money/2/ HTTP/1.1" 302 0 (2018-07-06 14:12:55; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 11054 (2018-07-06 14:12:55; basehttp.py:124)
INFO:django.server: "GET /18/ HTTP/1.1" 200 2910 (2018-07-06 14:13:00; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 11054 (2018-07-06 14:13:03; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-06 14:13:12; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-06 14:13:14; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-06 14:13:21; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 11062 (2018-07-06 14:13:21; basehttp.py:124)
INFO:django.server: "GET /add_money/1/ HTTP/1.1" 200 3107 (2018-07-06 14:13:24; basehttp.py:124)
INFO:django.server: "POST /add_money/1/ HTTP/1.1" 302 0 (2018-07-06 14:13:34; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 11058 (2018-07-06 14:13:34; basehttp.py:124)
INFO:django.server: "GET /18/ HTTP/1.1" 200 3226 (2018-07-06 14:13:38; basehttp.py:124)
INFO:django.server: "POST /set_executor/18/ HTTP/1.1" 302 0 (2018-07-06 14:13:40; basehttp.py:124)
INFO:django.server: "GET /18/ HTTP/1.1" 200 2978 (2018-07-06 14:13:40; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 11058 (2018-07-06 14:13:43; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-06 14:13:46; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-06 14:13:51; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-06 14:13:59; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 11054 (2018-07-06 14:13:59; basehttp.py:124)
INFO:django.server: "GET /18/ HTTP/1.1" 200 3219 (2018-07-06 14:14:04; basehttp.py:124)
INFO:django.server: "POST /done/18/ HTTP/1.1" 302 0 (2018-07-06 14:14:07; basehttp.py:124)
INFO:django.server: "GET /add_money/2/ HTTP/1.1" 200 3100 (2018-07-06 14:14:07; basehttp.py:124)
INFO:django.server: "POST /add_money/2/ HTTP/1.1" 302 0 (2018-07-06 14:14:27; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 11057 (2018-07-06 14:14:27; basehttp.py:124)
INFO:django.server: "GET /18/ HTTP/1.1" 200 3222 (2018-07-06 14:14:30; basehttp.py:124)
INFO:django.server: "POST /done/18/ HTTP/1.1" 302 0 (2018-07-06 14:14:33; basehttp.py:124)
INFO:django.server: "GET /18/ HTTP/1.1" 200 2902 (2018-07-06 14:14:33; basehttp.py:124)
INFO:django.server: "GET /add_money/2/ HTTP/1.1" 200 3100 (2018-07-06 14:14:42; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-06 14:14:44; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-06 14:14:46; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-06 14:14:52; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 11060 (2018-07-06 14:14:52; basehttp.py:124)
INFO:django.server: "GET /18/ HTTP/1.1" 200 2970 (2018-07-06 14:14:53; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 11060 (2018-07-06 14:15:20; basehttp.py:124)
INFO:django.server: "GET /18/ HTTP/1.1" 200 2970 (2018-07-06 14:15:50; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 11060 (2018-07-06 14:15:52; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 7293 (2018-07-06 14:15:53; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 6112 (2018-07-06 14:16:16; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 6960 (2018-07-06 14:18:03; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 6960 (2018-07-06 14:18:08; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 6960 (2018-07-06 14:18:10; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 4264 (2018-07-06 14:18:11; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 6960 (2018-07-06 14:18:14; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 4264 (2018-07-06 14:18:16; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 6960 (2018-07-06 14:18:18; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 4264 (2018-07-06 14:18:19; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 6960 (2018-07-06 14:18:19; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 4264 (2018-07-06 14:18:20; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 4082 (2018-07-06 14:18:33; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 4082 (2018-07-06 14:18:40; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 4082 (2018-07-06 14:20:14; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 4272 (2018-07-06 14:26:59; basehttp.py:124)
INFO:django.server: "GET /notice/new/ HTTP/1.1" 200 3247 (2018-07-06 14:27:02; basehttp.py:124)
INFO:django.server: "POST /notice/new/ HTTP/1.1" 302 0 (2018-07-06 14:27:08; basehttp.py:124)
INFO:django.server: "GET /19/ HTTP/1.1" 200 2913 (2018-07-06 14:27:08; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 11523 (2018-07-06 14:27:11; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 4561 (2018-07-06 14:27:12; basehttp.py:124)
INFO:django.server: "GET /notice/new/ HTTP/1.1" 200 3247 (2018-07-06 14:27:18; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 4561 (2018-07-06 14:27:19; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 5988 (2018-07-06 14:27:19; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 11523 (2018-07-06 14:28:05; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 4561 (2018-07-06 14:28:09; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 5988 (2018-07-06 14:28:10; basehttp.py:124)
ERROR:django.request: Internal Server Error: /your_tasks/ (2018-07-06 14:29:52; exception.py:135)
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 215, in _get_response
    response = response.render()
  File "/Library/Python/2.7/site-packages/django/template/response.py", line 107, in render
    self.content = self.rendered_content
  File "/Library/Python/2.7/site-packages/django/template/response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "/Library/Python/2.7/site-packages/django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 207, in render
    return self._render(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/Library/Python/2.7/site-packages/django/template/loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/Library/Python/2.7/site-packages/django/template/defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/Library/Python/2.7/site-packages/django/template/defaulttags.py", line 458, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/Library/Python/2.7/site-packages/django/urls/base.py", line 91, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Library/Python/2.7/site-packages/django/urls/resolvers.py", line 497, in _reverse_with_prefix
    raise NoReverseMatch(msg)
NoReverseMatch: Reverse for 'auth_update' not found. 'auth_update' is not a valid view function or pattern name.
ERROR:django.server: "GET /your_tasks/ HTTP/1.1" 500 161324 (2018-07-06 14:29:53; basehttp.py:124)
ERROR:django.request: Internal Server Error: /your_tasks/ (2018-07-06 14:41:38; exception.py:135)
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 215, in _get_response
    response = response.render()
  File "/Library/Python/2.7/site-packages/django/template/response.py", line 107, in render
    self.content = self.rendered_content
  File "/Library/Python/2.7/site-packages/django/template/response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "/Library/Python/2.7/site-packages/django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 207, in render
    return self._render(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/Library/Python/2.7/site-packages/django/template/loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/Library/Python/2.7/site-packages/django/template/defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/Library/Python/2.7/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/Library/Python/2.7/site-packages/django/template/defaulttags.py", line 458, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/Library/Python/2.7/site-packages/django/urls/base.py", line 91, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Library/Python/2.7/site-packages/django/urls/resolvers.py", line 497, in _reverse_with_prefix
    raise NoReverseMatch(msg)
NoReverseMatch: Reverse for 'auth_update' not found. 'auth_update' is not a valid view function or pattern name.
ERROR:django.server: "GET /your_tasks/ HTTP/1.1" 500 161324 (2018-07-06 14:41:38; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 5988 (2018-07-06 14:41:52; basehttp.py:124)
INFO:django.server: "GET /notice/new/ HTTP/1.1" 200 3247 (2018-07-06 14:41:54; basehttp.py:124)
INFO:django.server: "POST /notice/new/ HTTP/1.1" 302 0 (2018-07-06 14:41:59; basehttp.py:124)
INFO:django.server: "GET /20/ HTTP/1.1" 200 2911 (2018-07-06 14:41:59; basehttp.py:124)
WARNING:django.server: "GET /api HTTP/1.1" 403 4903 (2018-07-06 17:10:38; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927 (2018-07-06 17:10:39; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735 (2018-07-06 17:10:39; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131 (2018-07-06 17:10:39; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385 (2018-07-06 17:10:39; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045 (2018-07-06 17:10:39; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200 (2018-07-06 17:10:39; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817 (2018-07-06 17:10:39; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597 (2018-07-06 17:10:39; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632 (2018-07-06 17:10:39; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262 (2018-07-06 17:10:39; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458 (2018-07-06 17:10:39; basehttp.py:124)
INFO:django.server: "GET /api-auth/login/?next=/api HTTP/1.1" 200 2733 (2018-07-06 17:10:46; basehttp.py:124)
INFO:django.server: "POST /api-auth/login/ HTTP/1.1" 302 0 (2018-07-06 17:10:55; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 17:10:55; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 17:10:55; __init__.py:190)
INFO:django.server: "GET /api HTTP/1.1" 200 5611 (2018-07-06 17:10:55; basehttp.py:124)
INFO:django.server: "GET /api?format=json HTTP/1.1" 200 111 (2018-07-06 17:11:07; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 17:11:12; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 17:11:12; __init__.py:190)
INFO:django.server: "GET /api?format=api HTTP/1.1" 200 5699 (2018-07-06 17:11:12; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 17:11:14; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 17:11:14; __init__.py:190)
INFO:django.server: "OPTIONS /api?format=api HTTP/1.1" 200 5798 (2018-07-06 17:11:14; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 17:11:19; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 17:11:19; __init__.py:190)
INFO:django.server: "GET /api?format=api HTTP/1.1" 200 5699 (2018-07-06 17:11:19; basehttp.py:124)
INFO:django.server: "GET /apiusers HTTP/1.1" 301 0 (2018-07-06 17:11:28; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 17:11:28; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 17:11:28; __init__.py:190)
INFO:django.server: "GET /apiusers/ HTTP/1.1" 200 10030 (2018-07-06 17:11:28; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 17:12:00; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 17:12:00; __init__.py:190)
INFO:django.server: "POST /apiusers/ HTTP/1.1" 201 10179 (2018-07-06 17:12:00; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 17:12:11; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 17:12:11; __init__.py:190)
INFO:django.server: "GET /apiusers/ HTTP/1.1" 200 10363 (2018-07-06 17:12:11; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 17:12:16; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 17:12:16; __init__.py:190)
INFO:django.server: "GET /apiusers/ HTTP/1.1" 200 10363 (2018-07-06 17:12:16; basehttp.py:124)
INFO:django.server: "GET /api HTTP/1.1" 301 0 (2018-07-06 17:32:22; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 17:32:22; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 17:32:22; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5907 (2018-07-06 17:32:22; basehttp.py:124)
INFO:django.server: "GET /api/notices HTTP/1.1" 301 0 (2018-07-06 17:32:33; basehttp.py:124)
ERROR:django.request: Internal Server Error: /api/notices/ (2018-07-06 17:32:33; exception.py:135)
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Library/Python/2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/rest_framework/viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/Library/Python/2.7/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Library/Python/2.7/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/rest_framework/mixins.py", line 48, in list
    return Response(serializer.data)
  File "/Library/Python/2.7/site-packages/rest_framework/serializers.py", line 765, in data
    ret = super(ListSerializer, self).data
  File "/Library/Python/2.7/site-packages/rest_framework/serializers.py", line 262, in data
    self._data = self.to_representation(self.instance)
  File "/Library/Python/2.7/site-packages/rest_framework/serializers.py", line 683, in to_representation
    self.child.to_representation(item) for item in iterable
  File "/Library/Python/2.7/site-packages/django/db/models/query.py", line 250, in __iter__
    self._fetch_all()
  File "/Library/Python/2.7/site-packages/django/db/models/query.py", line 1102, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/Library/Python/2.7/site-packages/django/db/models/query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch)
  File "/Library/Python/2.7/site-packages/django/db/models/sql/compiler.py", line 863, in execute_sql
    sql, params = self.as_sql()
  File "/Library/Python/2.7/site-packages/django/db/models/sql/compiler.py", line 412, in as_sql
    extra_select, order_by, group_by = self.pre_sql_setup()
  File "/Library/Python/2.7/site-packages/django/db/models/sql/compiler.py", line 47, in pre_sql_setup
    order_by = self.get_order_by()
  File "/Library/Python/2.7/site-packages/django/db/models/sql/compiler.py", line 298, in get_order_by
    field, self.query.get_meta(), default_order=asc))
  File "/Library/Python/2.7/site-packages/django/db/models/sql/compiler.py", line 593, in find_ordering_name
    field, targets, alias, joins, path, opts = self._setup_joins(pieces, opts, alias)
  File "/Library/Python/2.7/site-packages/django/db/models/sql/compiler.py", line 626, in _setup_joins
    pieces, opts, alias)
  File "/Library/Python/2.7/site-packages/django/db/models/sql/query.py", line 1427, in setup_joins
    names, opts, allow_many, fail_on_missing=True)
  File "/Library/Python/2.7/site-packages/django/db/models/sql/query.py", line 1362, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
FieldError: Cannot resolve keyword 'pub_date' into field. Choices are: author, date_joined, email, first_name, groups, id, is_active, is_staff, is_superuser, last_login, last_name, logentry, notice, password, profile, registrationprofile, user_permissions, username
ERROR:django.server: "GET /api/notices/ HTTP/1.1" 500 154171 (2018-07-06 17:32:33; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 5017 (2018-07-06 17:43:00; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 5017 (2018-07-06 17:43:01; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/notice/ HTTP/1.1" 200 3301 (2018-07-06 17:43:12; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:43:12; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/notice/add/ HTTP/1.1" 200 7990 (2018-07-06 17:43:14; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:43:14; basehttp.py:124)
INFO:django.server: "POST /admin/exchange/notice/add/ HTTP/1.1" 302 0 (2018-07-06 17:43:37; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/notice/ HTTP/1.1" 200 4794 (2018-07-06 17:43:37; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:43:37; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/ HTTP/1.1" 200 2920 (2018-07-06 17:43:44; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 5354 (2018-07-06 17:43:46; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/ HTTP/1.1" 200 4716 (2018-07-06 17:43:47; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:43:47; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/4/change/ HTTP/1.1" 200 6127 (2018-07-06 17:43:50; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:43:50; basehttp.py:124)
INFO:django.server: "POST /admin/exchange/profile/4/change/ HTTP/1.1" 302 0 (2018-07-06 17:43:57; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/ HTTP/1.1" 200 4903 (2018-07-06 17:43:57; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:43:57; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/3/change/ HTTP/1.1" 200 6127 (2018-07-06 17:43:59; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:43:59; basehttp.py:124)
INFO:django.server: "POST /admin/exchange/profile/3/change/ HTTP/1.1" 302 0 (2018-07-06 17:44:01; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/ HTTP/1.1" 200 4903 (2018-07-06 17:44:02; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:44:02; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/ HTTP/1.1" 200 2920 (2018-07-06 17:44:03; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 5954 (2018-07-06 17:44:04; basehttp.py:124)
INFO:django.server: "GET /admin/auth/user/ HTTP/1.1" 200 7552 (2018-07-06 17:44:15; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:44:15; basehttp.py:124)
INFO:django.server: "GET /static/admin/img/icon-no.svg HTTP/1.1" 200 560 (2018-07-06 17:44:15; basehttp.py:124)
INFO:django.server: "GET /admin/auth/user/4/change/ HTTP/1.1" 200 15699 (2018-07-06 17:44:21; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/SelectBox.js HTTP/1.1" 200 5755 (2018-07-06 17:44:21; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/SelectFilter2.js HTTP/1.1" 200 12389 (2018-07-06 17:44:21; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:44:21; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/calendar.js HTTP/1.1" 200 7790 (2018-07-06 17:44:21; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 200 20529 (2018-07-06 17:44:21; basehttp.py:124)
INFO:django.server: "GET /static/admin/img/selector-icons.svg HTTP/1.1" 200 3291 (2018-07-06 17:44:21; basehttp.py:124)
INFO:django.server: "GET /static/admin/img/icon-unknown-alt.svg HTTP/1.1" 200 655 (2018-07-06 17:44:21; basehttp.py:124)
INFO:django.server: "GET /static/admin/img/icon-unknown.svg HTTP/1.1" 200 655 (2018-07-06 17:44:21; basehttp.py:124)
INFO:django.server: "GET /static/admin/img/icon-clock.svg HTTP/1.1" 200 677 (2018-07-06 17:44:21; basehttp.py:124)
INFO:django.server: "GET /static/admin/img/icon-calendar.svg HTTP/1.1" 200 1086 (2018-07-06 17:44:21; basehttp.py:124)
INFO:django.server: "POST /admin/auth/user/4/change/ HTTP/1.1" 302 0 (2018-07-06 17:45:11; basehttp.py:124)
INFO:django.server: "GET /admin/auth/user/ HTTP/1.1" 200 7744 (2018-07-06 17:45:11; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:45:11; basehttp.py:124)
INFO:django.server: "GET /admin/logout/ HTTP/1.1" 200 1147 (2018-07-06 17:45:15; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 302 0 (2018-07-06 17:45:17; basehttp.py:124)
INFO:django.server: "GET /admin/login/?next=/admin/ HTTP/1.1" 200 1740 (2018-07-06 17:45:17; basehttp.py:124)
INFO:django.server: "POST /admin/login/?next=/admin/ HTTP/1.1" 200 2011 (2018-07-06 17:45:24; basehttp.py:124)
INFO:django.server: "POST /admin/login/?next=/admin/ HTTP/1.1" 200 2011 (2018-07-06 17:45:30; basehttp.py:124)
INFO:django.server: "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0 (2018-07-06 17:45:40; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 6259 (2018-07-06 17:45:40; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3172 (2018-07-06 17:45:45; basehttp.py:124)
INFO:django.server: "GET /21/ HTTP/1.1" 200 2911 (2018-07-06 17:45:47; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-06 17:45:50; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-06 17:45:51; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 200 3127 (2018-07-06 17:45:58; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 302 0 (2018-07-06 17:46:03; basehttp.py:124)
INFO:django.server: "GET /admin/login/?next=/admin/ HTTP/1.1" 200 1740 (2018-07-06 17:46:03; basehttp.py:124)
INFO:django.server: "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0 (2018-07-06 17:46:11; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 6259 (2018-07-06 17:46:11; basehttp.py:124)
INFO:django.server: "GET /admin/registration/registrationprofile/ HTTP/1.1" 200 3766 (2018-07-06 17:46:15; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:46:15; basehttp.py:124)
INFO:django.server: "GET /admin/registration/registrationprofile/add/ HTTP/1.1" 200 4947 (2018-07-06 17:46:18; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:46:18; basehttp.py:124)
INFO:django.server: "GET /admin/auth/user/?_to_field=id&_popup=1 HTTP/1.1" 200 6222 (2018-07-06 17:46:48; basehttp.py:124)
INFO:django.server: "POST /admin/registration/registrationprofile/add/ HTTP/1.1" 302 0 (2018-07-06 17:47:01; basehttp.py:124)
INFO:django.server: "GET /admin/registration/registrationprofile/ HTTP/1.1" 200 5842 (2018-07-06 17:47:01; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:47:01; basehttp.py:124)
INFO:django.server: "GET /admin/registration/registrationprofile/1/change/ HTTP/1.1" 200 5283 (2018-07-06 17:47:05; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:47:05; basehttp.py:124)
INFO:django.server: "POST /admin/registration/registrationprofile/1/change/ HTTP/1.1" 302 0 (2018-07-06 17:47:07; basehttp.py:124)
INFO:django.server: "GET /admin/registration/registrationprofile/ HTTP/1.1" 200 5821 (2018-07-06 17:47:07; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:47:07; basehttp.py:124)
INFO:django.server: "POST /admin/registration/registrationprofile/ HTTP/1.1" 302 0 (2018-07-06 17:47:18; basehttp.py:124)
INFO:django.server: "GET /admin/registration/registrationprofile/ HTTP/1.1" 200 5818 (2018-07-06 17:47:18; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:47:19; basehttp.py:124)
INFO:django.server: "GET /static/admin/img/icon-alert.svg HTTP/1.1" 200 504 (2018-07-06 17:47:19; basehttp.py:124)
INFO:django.server: "POST /admin/registration/registrationprofile/ HTTP/1.1" 302 0 (2018-07-06 17:47:25; basehttp.py:124)
INFO:django.server: "GET /admin/registration/registrationprofile/ HTTP/1.1" 200 5562 (2018-07-06 17:47:25; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:47:25; basehttp.py:124)
ERROR:django.request: Internal Server Error: /accounts/activate/123/ (2018-07-06 17:47:35; exception.py:135)
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 215, in _get_response
    response = response.render()
  File "/Library/Python/2.7/site-packages/django/template/response.py", line 107, in render
    self.content = self.rendered_content
  File "/Library/Python/2.7/site-packages/django/template/response.py", line 82, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/Library/Python/2.7/site-packages/django/template/response.py", line 64, in resolve_template
    return select_template(template, using=self.using)
  File "/Library/Python/2.7/site-packages/django/template/loader.py", line 53, in select_template
    raise TemplateDoesNotExist(', '.join(template_name_list), chain=chain)
TemplateDoesNotExist: registration/activate.html
ERROR:django.server: "GET /accounts/activate/123/ HTTP/1.1" 500 81598 (2018-07-06 17:47:35; basehttp.py:124)
INFO:django.server: "GET /admin/registration/registrationprofile/ HTTP/1.1" 200 5562 (2018-07-06 17:47:44; basehttp.py:124)
INFO:django.server: "GET /admin/registration/registrationprofile/ HTTP/1.1" 200 5562 (2018-07-06 17:47:45; basehttp.py:124)
INFO:django.server: "GET /admin/registration/registrationprofile/ HTTP/1.1" 200 5562 (2018-07-06 17:47:45; basehttp.py:124)
INFO:django.server: "GET /admin/registration/registrationprofile/ HTTP/1.1" 200 5562 (2018-07-06 17:47:49; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:47:49; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3172 (2018-07-06 17:47:54; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-06 17:47:56; basehttp.py:124)
INFO:django.server: "GET /accounts/register/ HTTP/1.1" 200 3985 (2018-07-06 17:47:57; basehttp.py:124)
INFO:django.server: "POST /accounts/register/ HTTP/1.1" 302 0 (2018-07-06 17:48:13; basehttp.py:124)
INFO:django.server: "GET /accounts/register/complete/ HTTP/1.1" 200 2242 (2018-07-06 17:48:13; basehttp.py:124)
INFO:django.server: "GET /accounts/activate/InVzZXIxIg:1fbPDh:ced5ACRBsrcmZKB6ObC9HrRQ-IQ/ HTTP/1.1" 302 0 (2018-07-06 17:48:24; basehttp.py:124)
INFO:django.server: "GET /accounts/activate/complete/ HTTP/1.1" 200 2107 (2018-07-06 17:48:24; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-06 17:48:25; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-06 17:48:32; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3172 (2018-07-06 17:48:32; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 302 0 (2018-07-06 17:48:39; basehttp.py:124)
INFO:django.server: "GET /admin/login/?next=/admin/ HTTP/1.1" 200 2062 (2018-07-06 17:48:39; basehttp.py:124)
INFO:django.server: "POST /admin/login/?next=/admin/ HTTP/1.1" 200 2329 (2018-07-06 17:48:48; basehttp.py:124)
INFO:django.server: "GET /admin/login/?next=/admin/ HTTP/1.1" 200 2062 (2018-07-06 17:48:54; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-06 17:48:57; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-06 17:48:59; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-06 17:49:07; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3172 (2018-07-06 17:49:07; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 7000 (2018-07-06 17:49:10; basehttp.py:124)
INFO:django.server: "GET /admin/registration/registrationprofile/ HTTP/1.1" 200 5562 (2018-07-06 17:49:13; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:49:13; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 7000 (2018-07-06 17:49:16; basehttp.py:124)
INFO:django.server: "GET /admin/auth/user/ HTTP/1.1" 200 7981 (2018-07-06 17:49:22; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:49:22; basehttp.py:124)
INFO:django.server: "GET /admin/auth/user/5/change/ HTTP/1.1" 200 15947 (2018-07-06 17:49:24; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:49:24; basehttp.py:124)
INFO:django.server: "POST /admin/auth/user/5/change/ HTTP/1.1" 302 0 (2018-07-06 17:49:55; basehttp.py:124)
INFO:django.server: "GET /admin/auth/user/ HTTP/1.1" 200 8169 (2018-07-06 17:49:55; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:49:55; basehttp.py:124)
INFO:django.server: "GET /admin/logout/ HTTP/1.1" 200 1147 (2018-07-06 17:49:58; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 302 0 (2018-07-06 17:49:59; basehttp.py:124)
INFO:django.server: "GET /admin/login/?next=/admin/ HTTP/1.1" 200 1740 (2018-07-06 17:49:59; basehttp.py:124)
INFO:django.server: "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0 (2018-07-06 17:50:05; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 3212 (2018-07-06 17:50:05; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/notice/ HTTP/1.1" 200 4573 (2018-07-06 17:50:08; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:50:08; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 3212 (2018-07-06 17:50:12; basehttp.py:124)
INFO:django.server: "GET /admin/auth/ HTTP/1.1" 200 2464 (2018-07-06 17:50:14; basehttp.py:124)
INFO:django.server: "GET /admin/auth/ HTTP/1.1" 200 2464 (2018-07-06 17:50:16; basehttp.py:124)
INFO:django.server: "GET /admin/auth/ HTTP/1.1" 200 2464 (2018-07-06 17:50:18; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 3212 (2018-07-06 17:50:22; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/notice/ HTTP/1.1" 200 4573 (2018-07-06 17:50:24; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:50:24; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/notice/add/ HTTP/1.1" 200 6734 (2018-07-06 17:50:28; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:50:28; basehttp.py:124)
INFO:django.server: "POST /admin/exchange/notice/add/ HTTP/1.1" 302 0 (2018-07-06 17:50:40; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/notice/ HTTP/1.1" 200 5012 (2018-07-06 17:50:40; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:50:40; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/notice/22/change/ HTTP/1.1" 200 6973 (2018-07-06 17:50:42; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:50:42; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/notice/22/delete/ HTTP/1.1" 200 3363 (2018-07-06 17:50:44; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/cancel.js HTTP/1.1" 200 204 (2018-07-06 17:50:45; basehttp.py:124)
INFO:django.server: "POST /admin/exchange/notice/22/delete/ HTTP/1.1" 302 0 (2018-07-06 17:50:47; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/notice/ HTTP/1.1" 200 4740 (2018-07-06 17:50:47; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-06 17:50:47; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/ HTTP/1.1" 200 2488 (2018-07-06 17:50:50; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 3798 (2018-07-06 17:50:50; basehttp.py:124)
INFO:django.server: "GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 200 392 (2018-07-06 17:50:50; basehttp.py:124)
INFO:django.server: "GET /admin/logout/ HTTP/1.1" 200 1147 (2018-07-06 17:51:00; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 2419 (2018-07-06 17:51:04; basehttp.py:124)
INFO:django.server: "GET /accounts/register/ HTTP/1.1" 200 3985 (2018-07-06 17:51:06; basehttp.py:124)
INFO:django.server: "POST /accounts/register/ HTTP/1.1" 302 0 (2018-07-06 17:51:19; basehttp.py:124)
INFO:django.server: "GET /accounts/register/complete/ HTTP/1.1" 200 2242 (2018-07-06 17:51:19; basehttp.py:124)
INFO:django.server: "GET /accounts/activate/InVzZXIyIg:1fbPGh:fdUFqp9e6tUv5-Pd-kKBBcvrBzg/ HTTP/1.1" 302 0 (2018-07-06 17:51:28; basehttp.py:124)
INFO:django.server: "GET /accounts/activate/complete/ HTTP/1.1" 200 2107 (2018-07-06 17:51:28; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-06 17:51:29; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-06 17:51:41; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3172 (2018-07-06 17:51:41; basehttp.py:124)
INFO:django.server: "GET /21/ HTTP/1.1" 200 3223 (2018-07-06 17:51:42; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3172 (2018-07-06 17:51:43; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-06 17:51:46; basehttp.py:124)
INFO:django.server: "GET /accounts/register/ HTTP/1.1" 200 3985 (2018-07-06 17:51:47; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-06 17:51:48; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-06 17:51:54; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3172 (2018-07-06 17:51:54; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 7301 (2018-07-06 17:51:57; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3172 (2018-07-06 17:52:04; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-06 17:52:06; basehttp.py:124)
INFO:django.server: "GET /accounts/register/ HTTP/1.1" 200 3985 (2018-07-06 17:52:07; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-06 17:52:08; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-06 17:52:13; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3172 (2018-07-06 17:52:13; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 3798 (2018-07-06 17:52:18; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:01:19; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:01:19; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5907 (2018-07-06 18:01:19; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:01:49; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:01:49; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5907 (2018-07-06 18:01:49; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:01:52; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:01:52; __init__.py:190)
INFO:django.server: "GET /api/profiles/ HTTP/1.1" 200 11209 (2018-07-06 18:01:52; basehttp.py:124)
WARNING:django.server: "GET /api/ HTTP/1.1" 403 4909 (2018-07-06 18:03:25; basehttp.py:124)
WARNING:django.server: "GET /api/ HTTP/1.1" 403 4909 (2018-07-06 18:04:10; basehttp.py:124)
INFO:django.server: "GET /api-auth/login/?next=/api/ HTTP/1.1" 200 2734 (2018-07-06 18:04:11; basehttp.py:124)
INFO:django.server: "POST /api-auth/login/ HTTP/1.1" 302 0 (2018-07-06 18:04:17; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:04:17; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:04:17; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5907 (2018-07-06 18:04:17; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:05:42; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:05:42; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-06 18:05:42; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:05:45; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:05:45; __init__.py:190)
INFO:django.server: "GET /api/notices/ HTTP/1.1" 200 10382 (2018-07-06 18:05:45; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:06:04; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:06:04; __init__.py:190)
INFO:django.server: "POST /api/notices/ HTTP/1.1" 201 11079 (2018-07-06 18:06:04; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:06:12; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:06:12; __init__.py:190)
INFO:django.server: "OPTIONS /api/notices/ HTTP/1.1" 200 12833 (2018-07-06 18:06:12; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:06:23; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:06:23; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-06 18:06:23; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:06:31; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:06:31; __init__.py:190)
INFO:django.server: "GET /api/profiles/ HTTP/1.1" 200 9436 (2018-07-06 18:06:31; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:06:40; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:06:40; __init__.py:190)
WARNING:django.server: "POST /api/profiles/ HTTP/1.1" 400 9351 (2018-07-06 18:06:40; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:06:50; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:06:50; __init__.py:190)
WARNING:django.server: "POST /api/profiles/ HTTP/1.1" 400 9351 (2018-07-06 18:06:50; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:07:08; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:07:08; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-06 18:07:08; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:07:11; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:07:11; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 10138 (2018-07-06 18:07:11; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:07:19; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:07:19; __init__.py:190)
INFO:django.server: "POST /api/users/ HTTP/1.1" 201 10283 (2018-07-06 18:07:19; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:07:23; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:07:23; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 10469 (2018-07-06 18:07:23; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:07:25; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:07:25; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-06 18:07:25; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:07:27; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:07:27; __init__.py:190)
INFO:django.server: "GET /api/profiles/ HTTP/1.1" 200 9882 (2018-07-06 18:07:27; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:07:37; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:07:37; __init__.py:190)
WARNING:django.server: "POST /api/profiles/ HTTP/1.1" 400 9464 (2018-07-06 18:07:37; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:07:42; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:07:42; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-06 18:07:42; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:07:48; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:07:48; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 10469 (2018-07-06 18:07:48; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:07:51; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:07:51; __init__.py:190)
INFO:django.server: "OPTIONS /api/users/ HTTP/1.1" 200 11838 (2018-07-06 18:07:51; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:07:52; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:07:52; __init__.py:190)
INFO:django.server: "OPTIONS /api/users/ HTTP/1.1" 200 11838 (2018-07-06 18:07:52; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:07:53; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:07:53; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 10469 (2018-07-06 18:07:53; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:07:54; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:07:54; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 10469 (2018-07-06 18:07:54; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:07:56; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:07:56; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 10469 (2018-07-06 18:07:56; basehttp.py:124)
INFO:django.server: "GET /api/users/?format=json HTTP/1.1" 200 239 (2018-07-06 18:07:59; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-06 18:08:04; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-06 18:08:04; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-06 18:08:04; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:55:41; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:55:41; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 10469 (2018-07-09 09:55:41; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 304 0 (2018-07-09 09:55:41; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/css/default.css HTTP/1.1" 304 0 (2018-07-09 09:55:41; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/css/prettify.css HTTP/1.1" 304 0 (2018-07-09 09:55:41; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 304 0 (2018-07-09 09:55:41; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 304 0 (2018-07-09 09:55:41; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 304 0 (2018-07-09 09:55:41; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 304 0 (2018-07-09 09:55:41; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/csrf.js HTTP/1.1" 304 0 (2018-07-09 09:55:41; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/default.js HTTP/1.1" 304 0 (2018-07-09 09:55:41; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 304 0 (2018-07-09 09:55:41; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/img/grid.png HTTP/1.1" 304 0 (2018-07-09 09:55:42; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:55:45; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:55:45; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-09 09:55:45; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:55:47; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:55:47; __init__.py:190)
INFO:django.server: "GET /api/groups/ HTTP/1.1" 200 8663 (2018-07-09 09:55:47; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:56:05; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:56:05; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-09 09:56:05; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:56:07; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:56:07; __init__.py:190)
INFO:django.server: "GET /api/notices/ HTTP/1.1" 200 11177 (2018-07-09 09:56:07; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:56:25; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:56:25; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-09 09:56:25; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:56:27; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:56:27; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 10469 (2018-07-09 09:56:27; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:56:37; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:56:37; __init__.py:190)
INFO:django.server: "POST /api/users/ HTTP/1.1" 201 10281 (2018-07-09 09:56:37; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:56:46; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:56:46; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-09 09:56:46; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:56:48; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:56:48; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 10800 (2018-07-09 09:56:48; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:56:52; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:56:52; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-09 09:56:52; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:56:54; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:56:54; __init__.py:190)
INFO:django.server: "GET /api/profiles/ HTTP/1.1" 200 10324 (2018-07-09 09:56:54; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:57:11; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:57:11; __init__.py:190)
WARNING:django.server: "POST /api/profiles/ HTTP/1.1" 400 9572 (2018-07-09 09:57:11; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 5017 (2018-07-09 09:58:27; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/ HTTP/1.1" 200 4948 (2018-07-09 09:58:29; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-09 09:58:29; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/3/change/ HTTP/1.1" 200 6163 (2018-07-09 09:58:31; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-09 09:58:31; basehttp.py:124)
INFO:django.server: "POST /admin/exchange/profile/3/change/ HTTP/1.1" 302 0 (2018-07-09 09:58:37; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/ HTTP/1.1" 200 5135 (2018-07-09 09:58:37; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-09 09:58:37; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/3/change/ HTTP/1.1" 200 6165 (2018-07-09 09:58:39; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-09 09:58:39; basehttp.py:124)
INFO:django.server: "POST /admin/exchange/profile/3/change/ HTTP/1.1" 302 0 (2018-07-09 09:58:41; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/ HTTP/1.1" 200 5135 (2018-07-09 09:58:41; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-09 09:58:41; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3164 (2018-07-09 09:58:43; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-09 09:58:49; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-09 09:58:51; basehttp.py:124)
WARNING:django.server: "GET /api/profiles/ HTTP/1.1" 403 5090 (2018-07-09 09:59:14; basehttp.py:124)
WARNING:django.server: "GET /api/ HTTP/1.1" 403 4909 (2018-07-09 09:59:15; basehttp.py:124)
WARNING:django.server: "GET /api/users/ HTTP/1.1" 403 5054 (2018-07-09 09:59:16; basehttp.py:124)
WARNING:django.server: "GET /api/ HTTP/1.1" 403 4909 (2018-07-09 09:59:18; basehttp.py:124)
INFO:django.server: "GET /api-auth/login/?next=/api/ HTTP/1.1" 200 2734 (2018-07-09 09:59:20; basehttp.py:124)
INFO:django.server: "POST /api-auth/login/ HTTP/1.1" 200 3072 (2018-07-09 09:59:27; basehttp.py:124)
INFO:django.server: "POST /api-auth/login/ HTTP/1.1" 302 0 (2018-07-09 09:59:33; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:59:33; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:59:33; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-09 09:59:33; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 09:59:38; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 09:59:38; __init__.py:190)
INFO:django.server: "GET /api/notices/ HTTP/1.1" 200 11395 (2018-07-09 09:59:38; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2875 (2018-07-09 10:55:59; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3407 (2018-07-09 10:56:02; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 3081 (2018-07-09 10:56:04; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3407 (2018-07-09 10:56:04; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 3081 (2018-07-09 10:56:05; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3407 (2018-07-09 10:56:06; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 3081 (2018-07-09 10:56:07; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3407 (2018-07-09 10:56:07; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 3081 (2018-07-09 10:56:08; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3407 (2018-07-09 10:56:08; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 3081 (2018-07-09 10:56:09; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3407 (2018-07-09 10:56:10; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3407 (2018-07-09 10:56:15; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 3081 (2018-07-09 10:56:16; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3407 (2018-07-09 10:56:17; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3407 (2018-07-09 10:56:18; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 2904 (2018-07-09 10:56:45; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3164 (2018-07-09 10:56:47; basehttp.py:124)
INFO:django.server: "GET /notice/new/ HTTP/1.1" 200 3223 (2018-07-09 10:56:49; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3407 (2018-07-09 10:56:51; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 2904 (2018-07-09 10:56:58; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 2904 (2018-07-09 10:57:02; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3164 (2018-07-09 10:57:06; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 5644 (2018-07-09 11:23:34; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 5644 (2018-07-09 11:23:34; basehttp.py:124)
INFO:django.server: "GET /admin/auth/user/ HTTP/1.1" 200 7981 (2018-07-09 11:23:37; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-09 11:23:37; basehttp.py:124)
INFO:django.server: "GET /admin/auth/user/add/ HTTP/1.1" 200 6394 (2018-07-09 11:23:42; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-09 11:23:42; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 5644 (2018-07-09 11:23:51; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 11:23:54; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 11:23:54; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-09 11:23:54; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-09 11:23:56; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-09 11:23:56; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 10800 (2018-07-09 11:23:56; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3164 (2018-07-09 15:15:11; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 2904 (2018-07-09 15:15:16; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3164 (2018-07-09 15:15:17; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 2904 (2018-07-09 15:15:19; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3164 (2018-07-09 15:15:21; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-09 15:15:23; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-09 15:15:24; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 200 3127 (2018-07-09 15:15:30; basehttp.py:124)
INFO:django.server: "GET /accounts/register/ HTTP/1.1" 200 3985 (2018-07-09 15:15:32; basehttp.py:124)
INFO:django.server: "POST /accounts/register/ HTTP/1.1" 200 4184 (2018-07-09 15:15:46; basehttp.py:124)
INFO:django.server: "POST /accounts/register/ HTTP/1.1" 302 0 (2018-07-09 15:16:02; basehttp.py:124)
INFO:django.server: "GET /accounts/register/complete/ HTTP/1.1" 200 2242 (2018-07-09 15:16:02; basehttp.py:124)
INFO:django.server: "GET /accounts/activate/InJvb3RvdmljaDEyMyI:1fcSH4:BPYeC4dn7jqUnJ6ZKyWIBA8kySU/ HTTP/1.1" 302 0 (2018-07-09 15:16:14; basehttp.py:124)
INFO:django.server: "GET /accounts/activate/complete/ HTTP/1.1" 200 2107 (2018-07-09 15:16:14; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-09 15:16:18; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-09 15:16:26; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3171 (2018-07-09 15:16:26; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 3222 (2018-07-09 15:16:31; basehttp.py:124)
INFO:django.server: "POST /set_executor/1/ HTTP/1.1" 302 0 (2018-07-09 15:16:33; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 2975 (2018-07-09 15:16:33; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3171 (2018-07-09 15:16:35; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-09 15:18:25; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-09 15:18:27; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-09 15:18:33; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3164 (2018-07-09 15:18:33; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 3212 (2018-07-09 15:18:34; basehttp.py:124)
INFO:django.server: "POST /done/1/ HTTP/1.1" 302 0 (2018-07-09 15:18:35; basehttp.py:124)
INFO:django.server: "GET /add_money/1/ HTTP/1.1" 200 3085 (2018-07-09 15:18:35; basehttp.py:124)
INFO:django.server: "POST /add_money/1/ HTTP/1.1" 302 0 (2018-07-09 15:19:04; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3168 (2018-07-09 15:19:05; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 3216 (2018-07-09 15:19:05; basehttp.py:124)
INFO:django.server: "POST /done/1/ HTTP/1.1" 302 0 (2018-07-09 15:19:06; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 2897 (2018-07-09 15:19:06; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3167 (2018-07-09 15:19:07; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-09 15:19:14; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-09 15:19:16; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-09 15:19:22; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3173 (2018-07-09 15:19:22; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 2967 (2018-07-09 15:19:25; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 3302 (2018-07-09 15:19:27; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3122 (2018-07-09 15:19:29; basehttp.py:124)
INFO:django.server: "GET /notice/new/ HTTP/1.1" 200 3232 (2018-07-09 15:19:30; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3173 (2018-07-09 15:19:32; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 2411 (2018-07-13 10:08:15; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 2452 (2018-07-13 10:08:19; basehttp.py:124)
ERROR:django.request: Internal Server Error: /set_executor/1/ (2018-07-13 10:08:29; exception.py:135)
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/edit.py", line 240, in post
    return super(BaseUpdateView, self).post(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/edit.py", line 183, in post
    return self.form_valid(form)
  File "/Library/Python/2.7/site-packages/django/utils/decorators.py", line 185, in inner
    return func(*args, **kwargs)
  File "/Users/arthuratrokhov/Desktop/python/freelance/exchange/views.py", line 116, in form_valid
    form.instance.executor = self.request.user
  File "/Library/Python/2.7/site-packages/django/db/models/fields/related_descriptors.py", line 216, in __set__
    self.field.remote_field.model._meta.object_name,
ValueError: Cannot assign "<SimpleLazyObject: <django.contrib.auth.models.AnonymousUser object at 0x111a230d0>>": "Notice.executor" must be a "User" instance.
ERROR:django.server: "POST /set_executor/1/ HTTP/1.1" 500 94441 (2018-07-13 10:08:29; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-13 10:08:43; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 200 3127 (2018-07-13 10:08:51; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 200 3130 (2018-07-13 10:09:05; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 200 3123 (2018-07-13 10:09:13; basehttp.py:124)
INFO:django.server: "GET /accounts/register/ HTTP/1.1" 200 3985 (2018-07-13 10:09:15; basehttp.py:124)
INFO:django.server: "POST /accounts/register/ HTTP/1.1" 200 4160 (2018-07-13 10:09:30; basehttp.py:124)
INFO:django.server: "POST /accounts/register/ HTTP/1.1" 302 0 (2018-07-13 10:09:42; basehttp.py:124)
INFO:django.server: "GET /accounts/register/complete/ HTTP/1.1" 200 2242 (2018-07-13 10:09:42; basehttp.py:124)
INFO:django.server: "GET /accounts/activate/InVzZXIxMjMi:1fdpOo:WKNvmpgRaB7z4zn_1xyd1gjLpvw/ HTTP/1.1" 302 0 (2018-07-13 10:09:55; basehttp.py:124)
INFO:django.server: "GET /accounts/activate/complete/ HTTP/1.1" 200 2107 (2018-07-13 10:09:55; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-13 10:09:58; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-13 10:10:08; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3166 (2018-07-13 10:10:08; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 3207 (2018-07-13 10:10:09; basehttp.py:124)
INFO:django.server: "POST /set_executor/1/ HTTP/1.1" 302 0 (2018-07-13 10:10:11; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 2960 (2018-07-13 10:10:11; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3166 (2018-07-13 10:10:12; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-13 10:17:41; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-13 10:17:43; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-13 10:17:48; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3167 (2018-07-13 10:17:48; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 2903 (2018-07-13 10:17:50; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3167 (2018-07-13 10:17:53; basehttp.py:124)
INFO:django.server: "GET /1/ HTTP/1.1" 200 2903 (2018-07-13 10:19:05; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3167 (2018-07-13 10:19:06; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3167 (2018-07-13 10:19:07; basehttp.py:124)
WARNING:django.server: "GET /robots.txt HTTP/1.1" 404 3947 (2018-07-13 10:19:07; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3167 (2018-07-13 10:40:30; basehttp.py:124)
INFO:django.server: "GET /notice/new/ HTTP/1.1" 200 3226 (2018-07-13 10:40:32; basehttp.py:124)
INFO:django.server: "POST /notice/new/ HTTP/1.1" 302 0 (2018-07-13 10:40:38; basehttp.py:124)
INFO:django.server: "GET /2/ HTTP/1.1" 200 2907 (2018-07-13 10:40:38; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3612 (2018-07-13 10:40:43; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-13 10:40:49; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-13 10:40:50; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-13 10:41:00; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3611 (2018-07-13 10:41:00; basehttp.py:124)
INFO:django.server: "GET /2/ HTTP/1.1" 200 3217 (2018-07-13 10:41:03; basehttp.py:124)
INFO:django.server: "POST /set_executor/2/ HTTP/1.1" 302 0 (2018-07-13 10:41:04; basehttp.py:124)
INFO:django.server: "GET /2/ HTTP/1.1" 200 2970 (2018-07-13 10:41:04; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3611 (2018-07-13 10:41:06; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-13 10:41:09; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-13 10:41:11; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 200 3123 (2018-07-13 10:41:27; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-13 10:41:33; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3612 (2018-07-13 10:41:33; basehttp.py:124)
INFO:django.server: "GET /2/ HTTP/1.1" 200 3235 (2018-07-13 10:41:35; basehttp.py:124)
INFO:django.server: "POST /done/2/ HTTP/1.1" 302 0 (2018-07-13 10:41:36; basehttp.py:124)
INFO:django.server: "GET /2/ HTTP/1.1" 200 2889 (2018-07-13 10:41:36; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3612 (2018-07-13 10:41:37; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 3084 (2018-07-13 10:41:40; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3699 (2018-07-13 10:41:41; basehttp.py:124)
WARNING:django.server: "GET /robots.txt HTTP/1.1" 404 3947 (2018-07-13 11:13:43; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3699 (2018-07-13 11:13:43; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 3084 (2018-07-13 11:13:45; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-13 11:13:48; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-13 11:13:50; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-13 11:14:03; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3612 (2018-07-13 11:14:03; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3116 (2018-07-13 11:14:05; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 3497 (2018-07-13 11:14:06; basehttp.py:124)
WARNING:django.server: "GET /robots.txt HTTP/1.1" 404 3947 (2018-07-24 12:34:31; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 2856 (2018-07-24 12:34:31; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-24 12:34:32; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-24 12:34:39; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-24 12:34:45; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3612 (2018-07-24 12:34:45; basehttp.py:124)
INFO:django.server: "GET /admin HTTP/1.1" 301 0 (2018-07-24 12:34:51; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 5644 (2018-07-24 12:34:51; basehttp.py:124)
INFO:django.server: "GET /static/admin/css/base.css HTTP/1.1" 200 16066 (2018-07-24 12:34:51; basehttp.py:124)
INFO:django.server: "GET /static/admin/css/dashboard.css HTTP/1.1" 200 412 (2018-07-24 12:34:51; basehttp.py:124)
INFO:django.server: "GET /static/admin/css/fonts.css HTTP/1.1" 200 423 (2018-07-24 12:34:51; basehttp.py:124)
INFO:django.server: "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 81348 (2018-07-24 12:34:51; basehttp.py:124)
INFO:django.server: "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380 (2018-07-24 12:34:51; basehttp.py:124)
INFO:django.server: "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331 (2018-07-24 12:34:51; basehttp.py:124)
INFO:django.server: "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 80304 (2018-07-24 12:34:51; basehttp.py:124)
INFO:django.server: "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 82564 (2018-07-24 12:34:51; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/notice/ HTTP/1.1" 200 4791 (2018-07-24 12:35:01; basehttp.py:124)
INFO:django.server: "GET /static/admin/css/changelists.css HTTP/1.1" 200 6170 (2018-07-24 12:35:01; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-24 12:35:01; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/core.js HTTP/1.1" 200 7976 (2018-07-24 12:35:01; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 363 (2018-07-24 12:35:01; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 6591 (2018-07-24 12:35:01; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 258648 (2018-07-24 12:35:01; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/actions.js HTTP/1.1" 200 6501 (2018-07-24 12:35:01; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1538 (2018-07-24 12:35:01; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/urlify.js HTTP/1.1" 200 8186 (2018-07-24 12:35:01; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 128820 (2018-07-24 12:35:01; basehttp.py:124)
INFO:django.server: "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331 (2018-07-24 12:35:01; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 5644 (2018-07-24 12:35:05; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/notice/ HTTP/1.1" 200 4791 (2018-07-24 12:35:11; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-24 12:35:11; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 5644 (2018-07-24 12:36:20; basehttp.py:124)
INFO:django.server: "GET /admin/registration/registrationprofile/ HTTP/1.1" 200 3766 (2018-07-24 12:36:23; basehttp.py:124)
INFO:django.server: "GET /static/admin/img/search.svg HTTP/1.1" 200 458 (2018-07-24 12:36:23; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-24 12:36:23; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 5644 (2018-07-24 12:36:25; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/notice/ HTTP/1.1" 200 4791 (2018-07-24 12:36:25; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-24 12:36:26; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 5644 (2018-07-24 12:36:27; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/3/change/ HTTP/1.1" 200 6246 (2018-07-24 12:36:30; basehttp.py:124)
INFO:django.server: "GET /static/admin/css/forms.css HTTP/1.1" 200 8173 (2018-07-24 12:36:30; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/change_form.js HTTP/1.1" 200 712 (2018-07-24 12:36:30; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-24 12:36:30; basehttp.py:124)
INFO:django.server: "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 495 (2018-07-24 12:36:30; basehttp.py:124)
INFO:django.server: "GET /static/admin/css/widgets.css HTTP/1.1" 200 10340 (2018-07-24 12:36:30; basehttp.py:124)
INFO:django.server: "POST /admin/exchange/profile/3/change/ HTTP/1.1" 302 0 (2018-07-24 12:36:42; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/ HTTP/1.1" 200 5599 (2018-07-24 12:36:42; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-24 12:36:42; basehttp.py:124)
INFO:django.server: "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436 (2018-07-24 12:36:42; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/5/change/ HTTP/1.1" 200 6245 (2018-07-24 12:36:45; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-24 12:36:45; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/ HTTP/1.1" 200 5412 (2018-07-24 12:36:55; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/3/change/ HTTP/1.1" 200 6258 (2018-07-24 12:36:55; basehttp.py:124)
INFO:django.server: "GET /admin/exchange/profile/3/change/ HTTP/1.1" 200 6258 (2018-07-24 12:36:58; basehttp.py:124)
INFO:django.server: "GET /admin/jsi18n/ HTTP/1.1" 200 14962 (2018-07-24 12:36:58; basehttp.py:124)
INFO:django.server: "GET /admin/ HTTP/1.1" 200 5944 (2018-07-24 12:37:04; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3612 (2018-07-24 12:37:08; basehttp.py:124)
INFO:django.server: "GET /add_money/1/ HTTP/1.1" 200 3102 (2018-07-24 12:37:16; basehttp.py:124)
INFO:django.server: "POST /add_money/1/ HTTP/1.1" 302 0 (2018-07-24 12:37:18; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 3613 (2018-07-24 12:37:18; basehttp.py:124)
INFO:django.server: "GET /notice/new/ HTTP/1.1" 200 3227 (2018-07-24 12:37:23; basehttp.py:124)
INFO:django.server: "POST /notice/new/ HTTP/1.1" 302 0 (2018-07-24 12:37:41; basehttp.py:124)
INFO:django.server: "GET /3/ HTTP/1.1" 200 2922 (2018-07-24 12:37:41; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4068 (2018-07-24 12:37:44; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-24 12:37:47; basehttp.py:124)
WARNING:django.server: "GET /robots.txt HTTP/1.1" 404 3947 (2018-07-24 12:37:49; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-24 12:37:49; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 200 3130 (2018-07-24 12:37:58; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 200 3123 (2018-07-24 12:38:10; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-24 12:38:18; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4067 (2018-07-24 12:38:18; basehttp.py:124)
INFO:django.server: "GET /3/ HTTP/1.1" 200 3232 (2018-07-24 12:38:21; basehttp.py:124)
INFO:django.server: "POST /set_executor/3/ HTTP/1.1" 302 0 (2018-07-24 12:38:23; basehttp.py:124)
INFO:django.server: "GET /3/ HTTP/1.1" 200 2985 (2018-07-24 12:38:23; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-24 12:38:39; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-24 12:38:40; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-24 12:38:47; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4068 (2018-07-24 12:38:47; basehttp.py:124)
INFO:django.server: "GET /3/ HTTP/1.1" 200 3250 (2018-07-24 12:38:48; basehttp.py:124)
INFO:django.server: "POST /done/3/ HTTP/1.1" 302 0 (2018-07-24 12:38:54; basehttp.py:124)
INFO:django.server: "GET /add_money/1/ HTTP/1.1" 200 3103 (2018-07-24 12:38:54; basehttp.py:124)
INFO:django.server: "POST /add_money/1/ HTTP/1.1" 302 0 (2018-07-24 12:39:07; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4071 (2018-07-24 12:39:07; basehttp.py:124)
INFO:django.server: "GET /3/ HTTP/1.1" 200 3253 (2018-07-24 12:39:12; basehttp.py:124)
INFO:django.server: "POST /done/3/ HTTP/1.1" 302 0 (2018-07-24 12:39:13; basehttp.py:124)
INFO:django.server: "GET /3/ HTTP/1.1" 200 2906 (2018-07-24 12:39:13; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4070 (2018-07-24 12:39:21; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-24 12:39:57; basehttp.py:124)
INFO:django.server: "GET /accounts/register/ HTTP/1.1" 200 3985 (2018-07-24 12:39:58; basehttp.py:124)
WARNING:django.server: "GET /robots.txt HTTP/1.1" 404 3947 (2018-07-24 12:43:21; basehttp.py:124)
INFO:django.server: "POST /accounts/register/ HTTP/1.1" 302 0 (2018-07-24 12:43:22; basehttp.py:124)
INFO:django.server: "GET /accounts/register/complete/ HTTP/1.1" 200 2242 (2018-07-24 12:43:22; basehttp.py:124)
INFO:django.server: "GET /accounts/activate/InVzZXIxMTEi:1fhr2Y:b_ZDUilJDeC_r14C4onDDOUSi7s/ HTTP/1.1" 302 0 (2018-07-24 12:43:32; basehttp.py:124)
INFO:django.server: "GET /accounts/activate/complete/ HTTP/1.1" 200 2107 (2018-07-24 12:43:32; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-24 12:43:34; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-07-24 12:43:41; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4066 (2018-07-24 12:43:41; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4066 (2018-07-24 14:54:18; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4066 (2018-07-24 15:06:29; basehttp.py:124)
INFO:django.server: "GET /notice/new/ HTTP/1.1" 200 3225 (2018-07-24 15:32:53; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4066 (2018-07-24 15:32:56; basehttp.py:124)
INFO:django.server: "GET /3/ HTTP/1.1" 200 2902 (2018-07-24 15:32:57; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4066 (2018-07-24 15:33:00; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3115 (2018-07-24 15:33:01; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 3083 (2018-07-24 15:33:03; basehttp.py:124)
WARNING:django.server: "GET /robots.txt HTTP/1.1" 404 3947 (2018-07-24 15:33:04; basehttp.py:124)
INFO:django.server: "GET /notice/new/ HTTP/1.1" 200 3225 (2018-07-24 15:33:04; basehttp.py:124)
INFO:django.server: "POST /notice/new/ HTTP/1.1" 302 0 (2018-07-24 15:33:08; basehttp.py:124)
INFO:django.server: "GET /4/ HTTP/1.1" 200 2917 (2018-07-24 15:33:08; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4521 (2018-07-24 15:33:10; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3401 (2018-07-24 15:33:13; basehttp.py:124)
INFO:django.server: "GET /edit/4/ HTTP/1.1" 200 3258 (2018-07-24 15:33:15; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4521 (2018-07-24 15:33:36; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3401 (2018-07-24 15:33:52; basehttp.py:124)
INFO:django.server: "GET /your_tasks/ HTTP/1.1" 200 3083 (2018-07-24 15:33:57; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3401 (2018-07-24 15:34:05; basehttp.py:124)
INFO:django.server: "GET /notice/new/ HTTP/1.1" 200 3225 (2018-07-24 15:34:05; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4521 (2018-07-24 15:36:48; basehttp.py:124)
INFO:django.server: "GET /4/ HTTP/1.1" 200 2917 (2018-07-24 15:36:58; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4521 (2018-07-24 15:37:00; basehttp.py:124)
INFO:django.server: "GET /4/ HTTP/1.1" 200 2917 (2018-07-24 15:40:13; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-24 16:15:05; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-24 16:15:05; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-24 16:15:05; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-24 16:15:05; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-24 16:15:05; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-24 16:15:05; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-24 16:15:05; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-24 16:15:05; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-24 16:15:05; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-24 16:15:05; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-24 16:15:05; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-24 16:15:05; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-24 16:15:05; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-24 16:15:05; basehttp.py:124)
WARNING:django.server: "GET /robots.txt HTTP/1.1" 404 3947 (2018-07-26 11:57:24; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4521 (2018-07-26 11:57:24; basehttp.py:124)
INFO:django.server: "GET /api HTTP/1.1" 301 0 (2018-07-26 11:57:33; basehttp.py:124)
WARNING:django.server: "GET /api/ HTTP/1.1" 403 5150 (2018-07-26 11:57:33; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817 (2018-07-26 11:57:33; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735 (2018-07-26 11:57:33; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045 (2018-07-26 11:57:33; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632 (2018-07-26 11:57:33; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262 (2018-07-26 11:57:33; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385 (2018-07-26 11:57:33; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597 (2018-07-26 11:57:33; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927 (2018-07-26 11:57:33; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131 (2018-07-26 11:57:33; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200 (2018-07-26 11:57:33; basehttp.py:124)
INFO:django.server: "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458 (2018-07-26 11:57:33; basehttp.py:124)
INFO:django.server: "GET /api-auth/logout/?next=/api/ HTTP/1.1" 302 0 (2018-07-26 11:57:42; basehttp.py:124)
WARNING:django.server: "GET /api/ HTTP/1.1" 403 4909 (2018-07-26 11:57:42; basehttp.py:124)
INFO:django.server: "GET /api-auth/login/?next=/api/ HTTP/1.1" 200 2734 (2018-07-26 11:57:44; basehttp.py:124)
INFO:django.server: "POST /api-auth/login/ HTTP/1.1" 302 0 (2018-07-26 11:57:51; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-26 11:57:51; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-26 11:57:51; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-26 11:57:51; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-26 11:57:56; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-26 11:57:56; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 11796 (2018-07-26 11:57:56; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-26 12:00:16; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-26 12:00:16; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 12655 (2018-07-26 12:00:16; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-26 12:01:26; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-26 12:01:26; __init__.py:190)
INFO:django.server: "POST /api/users/ HTTP/1.1" 201 10640 (2018-07-26 12:01:26; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-26 12:01:56; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-26 12:01:56; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 13046 (2018-07-26 12:01:56; basehttp.py:124)
ERROR:django.request: Internal Server Error: /api/users/ (2018-07-26 12:02:44; exception.py:135)
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Library/Python/2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/rest_framework/viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/Library/Python/2.7/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Library/Python/2.7/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/rest_framework/mixins.py", line 48, in list
    return Response(serializer.data)
  File "/Library/Python/2.7/site-packages/rest_framework/serializers.py", line 765, in data
    ret = super(ListSerializer, self).data
  File "/Library/Python/2.7/site-packages/rest_framework/serializers.py", line 262, in data
    self._data = self.to_representation(self.instance)
  File "/Library/Python/2.7/site-packages/rest_framework/serializers.py", line 683, in to_representation
    self.child.to_representation(item) for item in iterable
  File "/Library/Python/2.7/site-packages/rest_framework/serializers.py", line 510, in to_representation
    fields = self._readable_fields
  File "/Library/Python/2.7/site-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/Library/Python/2.7/site-packages/rest_framework/serializers.py", line 376, in _readable_fields
    field for field in self.fields.values()
  File "/Library/Python/2.7/site-packages/rest_framework/serializers.py", line 363, in fields
    for key, value in self.get_fields().items():
  File "/Library/Python/2.7/site-packages/rest_framework/serializers.py", line 1045, in get_fields
    source, info, model, depth
  File "/Library/Python/2.7/site-packages/rest_framework/serializers.py", line 1190, in build_field
    return self.build_unknown_field(field_name, model_class)
  File "/Library/Python/2.7/site-packages/rest_framework/serializers.py", line 1302, in build_unknown_field
    (field_name, model_class.__name__)
ImproperlyConfigured: Field name `isactive` is not valid for model `User`.
ERROR:django.server: "GET /api/users/ HTTP/1.1" 500 147066 (2018-07-26 12:02:44; basehttp.py:124)
WARNING:django.server: "GET /robots.txt HTTP/1.1" 404 3947 (2018-07-26 12:03:38; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-26 12:03:38; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-26 12:03:38; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 13838 (2018-07-26 12:03:39; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4525 (2018-07-26 12:03:56; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-07-26 12:03:59; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-07-26 12:04:00; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 200 3130 (2018-07-26 12:04:09; basehttp.py:124)
WARNING:django.server: "GET /api/ HTTP/1.1" 403 4909 (2018-07-26 12:04:49; basehttp.py:124)
INFO:django.server: "GET /api-auth/login/?next=/api/ HTTP/1.1" 200 2734 (2018-07-26 12:04:52; basehttp.py:124)
INFO:django.server: "POST /api-auth/login/ HTTP/1.1" 302 0 (2018-07-26 12:04:58; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-26 12:04:58; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-26 12:04:58; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-26 12:04:58; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-26 12:05:09; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-26 12:05:09; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 13838 (2018-07-26 12:05:10; basehttp.py:124)
WARNING:django.server: "GET /robots.txt HTTP/1.1" 404 3947 (2018-07-26 13:26:38; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4525 (2018-07-26 13:26:38; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-26 13:26:47; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-26 13:26:47; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-26 13:26:47; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-26 13:27:08; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-26 13:27:08; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 13838 (2018-07-26 13:27:08; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-26 13:28:25; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-26 13:28:25; __init__.py:190)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-26 13:28:25; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-26 13:28:32; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-26 13:28:32; __init__.py:190)
INFO:django.server: "GET /api/notices/ HTTP/1.1" 200 14303 (2018-07-26 13:28:32; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-26 13:31:46; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-26 13:31:46; __init__.py:190)
INFO:django.server: "POST /api/notices/ HTTP/1.1" 201 12423 (2018-07-26 13:31:46; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4980 (2018-07-26 13:31:55; basehttp.py:124)
INFO:django.server: "GET /5/ HTTP/1.1" 200 2921 (2018-07-26 13:32:00; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4980 (2018-07-26 13:32:11; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-27 11:05:45; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-27 11:05:45; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-27 11:05:45; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-27 11:05:45; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-27 11:05:45; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-27 11:05:45; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-27 11:05:45; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-27 11:05:45; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-27 11:05:45; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-27 11:05:45; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-27 11:05:45; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-27 11:05:45; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-27 11:05:45; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-27 11:05:45; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-27 16:00:20; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-27 16:00:20; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-27 16:00:20; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-27 16:00:20; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-27 16:00:20; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-27 16:00:20; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-27 16:00:20; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-27 16:00:20; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-27 16:00:20; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-27 16:00:20; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-27 16:00:20; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-27 16:00:20; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-27 16:00:20; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-27 16:00:20; basehttp.py:124)
WARNING:django.server: "GET /robots.txt HTTP/1.1" 404 3947 (2018-07-27 17:22:37; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4980 (2018-07-27 17:22:37; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-27 17:22:43; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-27 17:22:43; __init__.py:190)
WARNING:django.server: "GET /robots.txt HTTP/1.1" 404 3947 (2018-07-27 17:22:43; basehttp.py:124)
INFO:django.server: "GET /api/ HTTP/1.1" 200 5899 (2018-07-27 17:22:43; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-27 17:22:46; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-27 17:22:46; __init__.py:190)
INFO:django.server: "GET /api/users/ HTTP/1.1" 200 13838 (2018-07-27 17:22:46; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-27 17:23:17; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-27 17:23:17; __init__.py:190)
INFO:django.server: "GET /api/notices/ HTTP/1.1" 200 14868 (2018-07-27 17:23:17; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-27 17:23:25; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-27 17:23:25; __init__.py:190)
INFO:django.server: "GET /api/users/5/ HTTP/1.1" 200 12670 (2018-07-27 17:23:26; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-27 17:23:33; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-27 17:23:33; __init__.py:190)
INFO:django.server: "GET /api/notices/1/ HTTP/1.1" 200 13962 (2018-07-27 17:23:33; basehttp.py:124)
DEBUG:MARKDOWN: Successfuly imported extension module "markdown.extensions.toc". (2018-07-27 17:23:41; __init__.py:233)
DEBUG:MARKDOWN: Successfully loaded extension "markdown.extensions.toc.TocExtension". (2018-07-27 17:23:41; __init__.py:190)
INFO:django.server: "GET /api/users/1/ HTTP/1.1" 200 12672 (2018-07-27 17:23:41; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-28 20:40:03; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-28 20:40:03; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-28 20:40:03; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-28 20:40:03; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-28 20:40:03; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-28 20:40:03; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-28 20:40:03; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-28 20:40:03; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-28 20:40:03; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-28 20:40:03; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-28 20:40:03; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-28 20:40:03; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-28 20:40:03; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-28 20:40:03; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-29 00:58:24; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-29 00:58:24; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-29 00:58:24; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-29 00:58:24; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-29 00:58:24; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-29 00:58:24; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-29 00:58:24; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-29 00:58:24; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-29 00:58:24; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-29 00:58:24; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-29 00:58:24; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-29 00:58:24; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-29 00:58:24; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-29 00:58:24; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-29 01:59:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-29 01:59:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-29 01:59:28; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-29 01:59:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-29 01:59:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-29 01:59:28; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-29 01:59:28; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-29 01:59:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-29 01:59:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-29 01:59:28; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-29 01:59:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-29 01:59:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-29 01:59:28; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-29 01:59:28; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-29 10:44:53; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-29 10:44:55; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-29 10:44:55; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-29 10:44:55; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-29 10:44:55; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-29 10:44:55; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-29 10:44:55; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-29 10:44:55; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-29 10:44:55; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-29 10:44:55; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-29 10:44:55; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-29 10:44:55; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-29 10:44:55; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-29 10:44:55; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-30 11:29:01; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-30 11:29:01; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-30 11:29:01; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-30 11:29:01; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-30 11:29:01; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-30 11:29:01; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-30 11:29:01; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-30 11:29:02; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-30 11:29:02; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-30 11:29:02; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-30 11:29:02; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-30 11:29:02; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-30 11:29:02; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-30 11:29:02; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-30 22:18:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-30 22:18:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-30 22:18:28; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-30 22:18:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-30 22:18:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-30 22:18:28; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-30 22:18:28; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-30 22:18:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-30 22:18:29; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-30 22:18:29; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-30 22:18:29; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-30 22:18:29; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-30 22:18:29; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-30 22:18:29; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-31 21:58:27; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-31 21:58:27; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-31 21:58:27; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-31 21:58:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-31 21:58:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-31 21:58:28; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-31 21:58:28; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-07-31 21:58:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-31 21:58:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-31 21:58:28; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-31 21:58:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-07-31 21:58:28; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-07-31 21:58:28; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-07-31 21:58:28; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-08-01 01:12:09; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-08-01 01:12:09; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-08-01 01:12:09; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-08-01 01:12:09; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-08-01 01:12:09; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-08-01 01:12:09; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-08-01 01:12:09; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-08-01 01:12:09; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-08-01 01:12:09; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-08-01 01:12:09; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-08-01 01:12:09; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-08-01 01:12:09; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-08-01 01:12:09; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-08-01 01:12:09; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-08-01 10:32:14; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-08-01 10:32:14; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-08-01 10:32:14; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-08-01 10:32:14; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-08-01 10:32:14; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-08-01 10:32:14; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-08-01 10:32:14; basehttp.py:124)
WARNING:django.server: "GET /converter/ HTTP/1.1" 404 3947 (2018-08-01 10:32:14; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-08-01 10:32:14; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-08-01 10:32:14; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-08-01 10:32:14; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 4013 (2018-08-01 10:32:14; basehttp.py:124)
WARNING:django.server: "GET /apple-touch-icon.png HTTP/1.1" 404 3977 (2018-08-01 10:32:14; basehttp.py:124)
WARNING:django.server: "GET /favicon.ico HTTP/1.1" 404 3950 (2018-08-01 10:32:14; basehttp.py:124)
WARNING:django.server: "GET /robots.txt HTTP/1.1" 404 3947 (2018-08-01 10:49:45; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 4980 (2018-08-01 10:49:45; basehttp.py:124)
INFO:django.server: "GET /5/ HTTP/1.1" 200 2921 (2018-08-01 10:49:49; basehttp.py:124)
INFO:django.server: "GET /notice/new/ HTTP/1.1" 200 3229 (2018-08-01 10:49:50; basehttp.py:124)
ERROR:django.request: Internal Server Error: /notice/new/ (2018-08-01 10:49:56; exception.py:135)
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/edit.py", line 183, in post
    return self.form_valid(form)
  File "/Library/Python/2.7/site-packages/django/utils/decorators.py", line 185, in inner
    return func(*args, **kwargs)
  File "/Users/arthuratrokhov/Desktop/python/freelance/exchange/views.py", line 58, in form_valid
    form.instance.author = User.objects.select_for_update().get(id=request.user.id)
NameError: global name 'request' is not defined
ERROR:django.server: "POST /notice/new/ HTTP/1.1" 500 89427 (2018-08-01 10:49:56; basehttp.py:124)
ERROR:django.request: Internal Server Error: /notice/new/ (2018-08-01 10:50:36; exception.py:135)
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/edit.py", line 183, in post
    return self.form_valid(form)
  File "/Library/Python/2.7/site-packages/django/utils/decorators.py", line 185, in inner
    return func(*args, **kwargs)
  File "/Users/arthuratrokhov/Desktop/python/freelance/exchange/views.py", line 58, in form_valid
    form.instance.author = User.objects.select_for_update().get(id=request.user.id)
NameError: global name 'request' is not defined
ERROR:django.server: "POST /notice/new/ HTTP/1.1" 500 89427 (2018-08-01 10:50:37; basehttp.py:124)
ERROR:django.request: Internal Server Error: /notice/new/ (2018-08-01 10:51:15; exception.py:135)
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/edit.py", line 183, in post
    return self.form_valid(form)
  File "/Library/Python/2.7/site-packages/django/utils/decorators.py", line 185, in inner
    return func(*args, **kwargs)
  File "/Users/arthuratrokhov/Desktop/python/freelance/exchange/views.py", line 58, in form_valid
    form.instance.author = User.objects.select_for_update().get(id=self.request.user)
  File "/Library/Python/2.7/site-packages/django/db/models/query.py", line 370, in get
    clone = self.filter(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/db/models/query.py", line 781, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/db/models/query.py", line 799, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/Library/Python/2.7/site-packages/django/db/models/sql/query.py", line 1260, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/Library/Python/2.7/site-packages/django/db/models/sql/query.py", line 1286, in _add_q
    allow_joins=allow_joins, split_subq=split_subq,
  File "/Library/Python/2.7/site-packages/django/db/models/sql/query.py", line 1220, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "/Library/Python/2.7/site-packages/django/db/models/sql/query.py", line 1114, in build_lookup
    return final_lookup(lhs, rhs)
  File "/Library/Python/2.7/site-packages/django/db/models/lookups.py", line 24, in __init__
    self.rhs = self.get_prep_lookup()
  File "/Library/Python/2.7/site-packages/django/db/models/lookups.py", line 74, in get_prep_lookup
    return self.lhs.output_field.get_prep_value(self.rhs)
  File "/Library/Python/2.7/site-packages/django/db/models/fields/__init__.py", line 962, in get_prep_value
    return int(value)
TypeError: int() argument must be a string or a number, not 'SimpleLazyObject'
ERROR:django.server: "POST /notice/new/ HTTP/1.1" 500 135230 (2018-08-01 10:51:16; basehttp.py:124)
INFO:django.server: "GET /notice/new/ HTTP/1.1" 200 3229 (2018-08-01 10:51:22; basehttp.py:124)
ERROR:django.request: Internal Server Error: /notice/new/ (2018-08-01 10:51:27; exception.py:135)
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/edit.py", line 183, in post
    return self.form_valid(form)
  File "/Library/Python/2.7/site-packages/django/utils/decorators.py", line 185, in inner
    return func(*args, **kwargs)
  File "/Users/arthuratrokhov/Desktop/python/freelance/exchange/views.py", line 58, in form_valid
    form.instance.author = User.objects.select_for_update().get(id=self.request.user)
  File "/Library/Python/2.7/site-packages/django/db/models/query.py", line 370, in get
    clone = self.filter(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/db/models/query.py", line 781, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/db/models/query.py", line 799, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/Library/Python/2.7/site-packages/django/db/models/sql/query.py", line 1260, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/Library/Python/2.7/site-packages/django/db/models/sql/query.py", line 1286, in _add_q
    allow_joins=allow_joins, split_subq=split_subq,
  File "/Library/Python/2.7/site-packages/django/db/models/sql/query.py", line 1220, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "/Library/Python/2.7/site-packages/django/db/models/sql/query.py", line 1114, in build_lookup
    return final_lookup(lhs, rhs)
  File "/Library/Python/2.7/site-packages/django/db/models/lookups.py", line 24, in __init__
    self.rhs = self.get_prep_lookup()
  File "/Library/Python/2.7/site-packages/django/db/models/lookups.py", line 74, in get_prep_lookup
    return self.lhs.output_field.get_prep_value(self.rhs)
  File "/Library/Python/2.7/site-packages/django/db/models/fields/__init__.py", line 962, in get_prep_value
    return int(value)
TypeError: int() argument must be a string or a number, not 'SimpleLazyObject'
ERROR:django.server: "POST /notice/new/ HTTP/1.1" 500 135228 (2018-08-01 10:51:27; basehttp.py:124)
INFO:django.server: "POST /notice/new/ HTTP/1.1" 302 0 (2018-08-01 10:54:10; basehttp.py:124)
INFO:django.server: "GET /6/ HTTP/1.1" 200 2917 (2018-08-01 10:54:10; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 5427 (2018-08-01 10:54:13; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 5427 (2018-08-01 10:59:40; basehttp.py:124)
INFO:django.server: "GET /notice/new/ HTTP/1.1" 200 3229 (2018-08-01 10:59:42; basehttp.py:124)
INFO:django.server: "POST /notice/new/ HTTP/1.1" 302 0 (2018-08-01 10:59:50; basehttp.py:124)
INFO:django.server: "GET /7/ HTTP/1.1" 200 2922 (2018-08-01 10:59:50; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 5878 (2018-08-01 10:59:53; basehttp.py:124)
INFO:django.server: "GET /7/ HTTP/1.1" 200 2922 (2018-08-01 11:00:00; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 5878 (2018-08-01 11:00:02; basehttp.py:124)
INFO:django.server: "GET /7/ HTTP/1.1" 200 2922 (2018-08-01 11:00:31; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 5878 (2018-08-01 11:00:33; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 4876 (2018-08-01 11:00:33; basehttp.py:124)
WARNING:django.server: "GET /robots.txt HTTP/1.1" 404 3947 (2018-08-01 11:00:36; basehttp.py:124)
INFO:django.server: "GET /edit/7/ HTTP/1.1" 200 3260 (2018-08-01 11:00:36; basehttp.py:124)
INFO:django.server: "POST /edit/7/ HTTP/1.1" 302 0 (2018-08-01 11:00:39; basehttp.py:124)
INFO:django.server: "GET /7/ HTTP/1.1" 200 2923 (2018-08-01 11:00:39; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 5878 (2018-08-01 11:00:40; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 5878 (2018-08-01 11:10:42; basehttp.py:124)
INFO:django.server: "GET /add_money/1/ HTTP/1.1" 200 3108 (2018-08-01 11:17:25; basehttp.py:124)
ERROR:django.request: Internal Server Error: /add_money/1/ (2018-08-01 11:17:31; exception.py:135)
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/edit.py", line 240, in post
    return super(BaseUpdateView, self).post(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/edit.py", line 183, in post
    return self.form_valid(form)
  File "/Library/Python/2.7/site-packages/django/utils/decorators.py", line 185, in inner
    return func(*args, **kwargs)
  File "/Users/arthuratrokhov/Desktop/python/freelance/exchange/views.py", line 99, in form_valid
    form.instance.update(current_balance = F('current_balance') + form.instance.transaction_sum)
AttributeError: 'Profile' object has no attribute 'update'
ERROR:django.server: "POST /add_money/1/ HTTP/1.1" 500 89732 (2018-08-01 11:17:31; basehttp.py:124)
ERROR:django.request: Internal Server Error: /add_money/1/ (2018-08-01 11:17:52; exception.py:135)
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Library/Python/2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/edit.py", line 240, in post
    return super(BaseUpdateView, self).post(request, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/django/views/generic/edit.py", line 183, in post
    return self.form_valid(form)
  File "/Library/Python/2.7/site-packages/django/utils/decorators.py", line 185, in inner
    return func(*args, **kwargs)
  File "/Users/arthuratrokhov/Desktop/python/freelance/exchange/views.py", line 99, in form_valid
    form.instance.current_balance = F('current_balance') + form.instance.transaction_sum
NameError: global name 'F' is not defined
ERROR:django.server: "POST /add_money/1/ HTTP/1.1" 500 89556 (2018-08-01 11:17:52; basehttp.py:124)
INFO:django.server: "POST /add_money/1/ HTTP/1.1" 302 0 (2018-08-01 11:18:58; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 5878 (2018-08-01 11:18:58; basehttp.py:124)
INFO:django.server: "GET /add_money/1/ HTTP/1.1" 200 3101 (2018-08-01 11:19:02; basehttp.py:124)
INFO:django.server: "POST /add_money/1/ HTTP/1.1" 302 0 (2018-08-01 11:19:03; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 5878 (2018-08-01 11:19:03; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 5878 (2018-08-01 11:20:20; basehttp.py:124)
INFO:django.server: "GET /4/ HTTP/1.1" 200 3232 (2018-08-01 11:20:25; basehttp.py:124)
INFO:django.server: "POST /set_executor/4/ HTTP/1.1" 302 0 (2018-08-01 11:20:26; basehttp.py:124)
INFO:django.server: "GET /4/ HTTP/1.1" 200 2985 (2018-08-01 11:20:26; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 5878 (2018-08-01 11:20:28; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-08-01 11:20:32; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-08-01 11:20:34; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-08-01 11:20:44; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 5874 (2018-08-01 11:20:44; basehttp.py:124)
INFO:django.server: "GET /your_notices/ HTTP/1.1" 200 3402 (2018-08-01 11:20:49; basehttp.py:124)
INFO:django.server: "GET /4/ HTTP/1.1" 200 3245 (2018-08-01 11:20:53; basehttp.py:124)
INFO:django.server: "POST /done/4/ HTTP/1.1" 302 0 (2018-08-01 11:21:24; basehttp.py:124)
INFO:django.server: "GET /add_money/6/ HTTP/1.1" 200 3087 (2018-08-01 11:21:24; basehttp.py:124)
INFO:django.server: "POST /add_money/6/ HTTP/1.1" 302 0 (2018-08-01 11:21:31; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 5876 (2018-08-01 11:21:31; basehttp.py:124)
INFO:django.server: "GET /4/ HTTP/1.1" 200 3247 (2018-08-01 11:21:34; basehttp.py:124)
WARNING:django.server: "GET /robots.txt HTTP/1.1" 404 3947 (2018-08-01 11:21:36; basehttp.py:124)
INFO:django.server: "POST /done/4/ HTTP/1.1" 302 0 (2018-08-01 11:21:36; basehttp.py:124)
INFO:django.server: "GET /4/ HTTP/1.1" 200 2901 (2018-08-01 11:21:36; basehttp.py:124)
INFO:django.server: "GET /accounts/logout/ HTTP/1.1" 200 2024 (2018-08-01 11:21:47; basehttp.py:124)
INFO:django.server: "GET /accounts/login/ HTTP/1.1" 200 2872 (2018-08-01 11:21:48; basehttp.py:124)
INFO:django.server: "POST /accounts/login/ HTTP/1.1" 302 0 (2018-08-01 11:22:14; basehttp.py:124)
INFO:django.server: "GET / HTTP/1.1" 200 5878 (2018-08-01 11:22:14; basehttp.py:124)
